packages/core/src/shared/sam/utils.ts

```diff
@@ -19,6 +25,23 @@ import { RuntimeFamily, getFamily } from '../../lambda/models/samLambdaRuntime'
 export const getProjectRoot = (template: TemplateItem | undefined) =>
     template ? getProjectRootUri(template.uri) : undefined
 
+/**
+ * @description look for template in project root
```

Comment:
"template" could mean anything. There are all kinds of templates. It's very confusing for developers to know when it is referring to SAM templates. That's why I also specify template.yaml.

Also, `@description` is not needed.
```suggestion
 * Find template.yaml in project root
```

Reply:
Acknowledge. I'll update this.